src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -241,6 +241,10 @@ export class CellModel extends Disposable implements ICellModel {
 			const newEditMode = this._lastEditMode ?? this._defaultTextEditMode;
 			this.showPreview = newEditMode !== TextCellEditModes.Markdown;
 			this.showMarkdown = newEditMode !== TextCellEditModes.RichText;
+		} else {
+			// when not in edit mode, default the values to false.
```

Comment:
This isn't really helpful - it's clear enough that you're setting the values to false. Better to explain WHY you're setting them to false.

Reply:
They are only valid when editing because the modes exist only when editing, so the why is understood from the first part, doesn't it?

Reply:
Well - I don't think that someone not familiar with the code would understand the interaction between "isEditMode" and showPreview/showMarkdown. 

The general point is that if you're adding a comment at all you should make sure it actually provides helpful information - and typically that is going to be "why is this being done".

Reply:
Also FYI - please make sure you update the title/description when you make changes like this. Just reading those as they are it's not apparent that you're actually fixing the behavior of the cell logic and it's important that you call that out for people reviewing this (and explain what you're fixing and why)